src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceServiceSpec.scala

```diff
@@ -42,7 +42,7 @@ class WorkspaceServiceSpec extends FreeSpec with ScalaFutures with ScalatestRout
       name = Option.empty,
       namespace = Option.empty)
 
-    val validWorkspacePath = FireCloudConfig.Workspace.importEntitiesPathFromWorkspace(
+    val tsvImportPath = ApiPrefix + "/%s/%s/importEntities".format(
```

Comment:
Fixed this as per conversation with @rushtong: the config paths specify endpoints for *external* services, not our own endpoints.